### PR TITLE
Minor updates & corrections in supply chain in sdk docs 3.0.2 - Closes #893

### LINKED ISSUE #893: Minor updates & corrections in Supply chain in SDK docs 3.0.2
Parent issue #891

### PR DESCRIPTION
**Pull request by chris529**
Close #893

### CODE REVIEW

File: modules/ROOT/pages/tutorials/supply-chain/part4.adoc [hunk 1]
```diff
@@ -67,11 +67,11 @@ During development it is both necessary and convenient to have a rather centrali
 
 Once the development has reached the point of concept or a usable product stage, it will be necessary to add more nodes to the network, and hence offer other potential users the opportunity to join the newly created blockchain network.
 
-After completion of part 4, the set up should appear as shown below in the following diagram: xref:{url_tutorials_transport_2}[Part 3: A simple supply chain tracking system]:
+After completion of part 4, the set up should appear as shown below in the following diagram: xref:{url_tutorials_transport_2}[Part 3: A simple supply chain tracking system].
 
 image:launch-seed_node.png[One node diagram]
 
-The next step is to add one more node to the network that communicates between the seed node, the IoT and the client application as shown in the next diagram below:
+The next step is to add one more node to the network that communicates between the seed node, the IoT, and the client application as shown in the next diagram below:
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/supply-chain/part4.adoc]: Please never add any commas before `and`

**Review by Tschakki — CHANGES_REQUESTED** [review]
Hi Chris, I saw several times you add a comma before `and`, please stop adding it, it is wrong grammar. There are only very specific cases, where a comma before `and` makes sense.

File: modules/ROOT/pages/tutorials/supply-chain/part4.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/tutorials/supply-chain/part4.adoc]: OK, it should now be removed.

Comment (Tschakki) [reply modules/ROOT/pages/tutorials/supply-chain/part4.adoc]: did you push the changes? I still see it.

Comment (chris529) [reply modules/ROOT/pages/tutorials/supply-chain/part4.adoc]: It should work now.. 👍

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**